File: genefunk-starter/src/main/java/de/hipp/pnp/GeneFunkServiceApplication.java
package de.hipp.pnp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "de.hipp.*")
public class GeneFunkServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(GeneFunkServiceApplication.class, args);
    }

}

File: genefunk-starter/src/main/kotlin/de/hipp/pnp/GeneFunkServiceApplication.kt
package de.hipp.pnp

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication(scanBasePackages = ["de.hipp.*"])
open class GeneFunkServiceApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(GeneFunkServiceApplication::class.java, *args)
        }
    }
}
